smart-home-provider/cloud/smart-home-provider-cloud.js

```diff
@@ -593,31 +593,37 @@ const server = app.listen(appPort, () => {
   console.log('Smart Home Cloud and App listening at %s:%s', host, port);
 
   if (config.isLocal) {
-    ngrok.connect(config.devPortSmartHome, (err, url) => {
+    localtunnel(config.devPortSmartHome, {subdomain: config.devSubdomainSmartHome}, (err, tunnel) => {
       if (err) {
-        console.log('ngrok err', err);
+        console.log('localtunnel err', err);
+        process.exit();
+      }
+
+      if (tunnel.url !== 'https://' + config.devSubdomainSmartHome + '.localtunnel.me') {
+        console.log('localtunnel subdomain already in use, specify a different one in' +
+          ' config-provider.js', config.devSubdomainSmartHome);
         process.exit();
       }
 
       console.log('|###################################################|');
       console.log('|                                                   |');
-      console.log('|        COPY & PASTE NGROK URL BELOW:              |');
+      console.log('|     COPY & PASTE LOCALTUNNEL URL BELOW:           |');
       console.log('|                                                   |');
-      console.log('|          ' + url + '                |');
+      console.log('|     ' + tunnel.url + '        |');
```

Comment:
Can we guarantee the length of this will always be the same? Maybe we should redesign this logging

Reply:
We can't guarantee the length since it's arbitrary. I've simplified this block of log output so it looks similar to the following ones